code/game/objects/items/weapons/batons.dm

```diff
@@ -16,8 +16,12 @@
 	// Settings
 	/// Whether the baton can stun silicon mobs
 	var/affect_silicon = FALSE
+	/// The amount of stamina damage the baton does per swing
+	var/stamina_damage = 30
+	/// The amount the stamina damage ignores armour
```

Comment:
```suggestion
	/// How much stamina armor is ignored by the stamina damage
```

Reply:
`stamina armor` isn't a thing. it goes off of melee armour

Reply:
"how much of *melee armor* is ignored by the stamina damage" then 🙃